tldw_Server_API/tests/TTS_NEW/unit/test_tokenizer_service_qwen3.py

```
@@ -47,30 +47,34 @@ def test_serialize_audio_output_wav_passthrough():
-    original_fn = tokenizer_cls.chunked_decode
+def test_load_qwen3_tokenizer_uses_from_pretrained_when_available(monkeypatch):
+    module = types.ModuleType("qwen_tts")
 
-    tokenizer = tokenizer_service._load_qwen3_tokenizer(
-        "Qwen/Qwen3-TTS-Tokenizer-12Hz/",
-        allow_download=False,
-        patch_chunked_decode=False,
-    )
+    class FakeTokenizer:
+        @classmethod
+        def from_pretrained(cls, model_id, local_files_only=True):
```

Ruff (0.15.7)

[warning] Missing return type annotation for classmethod `from_pretrained`

(ANN206)